scripts/gobuild.sh

```diff
@@ -20,7 +20,7 @@ export GOPATH="${TOPWD}/ecs-init/:${BUILDDIR}"
 export SRCPATH="${BUILDDIR}/src/github.com/aws/amazon-ecs-init"
 
 if [ -d "${TOPWD}/.git" ]; then
-    version=$(cat "${TOPWD}/ecs-init/VERSION")
+    version=$(cat "${TOPWD}/ecs-init/ECSVERSION")
```

Comment:
I think I've heard peeps of discontent around this before, thanks for fixing this!